grunt/scsslint.js

```diff
@@ -1,20 +0,0 @@
-/*
- * SCSS Lint Grunt plugin
- * https://github.com/ahmednuaman/grunt-scss-lint
- */
-
-module.exports = {
-
-    // Lint scss folder
-    scss: [
-        '<%= srcPath %>components/**/*.scss',
-        '!<%= srcPath %>components/sass/vendor/**/*.scss'
-    ],
-
-    options: {
-        config: '.scss-lint.yml',
-        reporterOutput: '<%= testResultPath %>scss-lint-report.xml',
-        compact: true,
-        maxBuffer: NaN
-    }
-};
```

Comment:
Removing `scss-lint` would be a breaking change. We are developing for 2.x here, so no breaking changes. Please revert the removal of `scss-lint`.

Reply:
By merging branch feature/add-stylelint to feature/watch-task plugins grunt-contrib-csslint and grunt-scss-lint were removed. Since this is a breaking change I took both plugins in again.